src/lib/components/Paper.react.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React, { Component } from 'react';
+
+import MuiPaper from '@material-ui/core/Paper';
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+type Props = {
+  /** ID for Paper */
+  id?: string,
+  /** Can be used to render elements inside the Paper*/
+  children?: Node,
+  /** The CSS class name of the root element */
+  className?: string,
+  /** The classes to be applied to this component. This keys in this object must be valid CSS rule
+   * names, and the values must be strings for the classnames to be assigned to each rule name
+   * Valid rule names are:
+   *   root
+   *   rounded
+   *   outlined
+   *   elevation0
+   *   elevation1
+   *   ...
+   *   elevation23
+   *   elevation24
+   */
+  classes?: {
+    root?: string,
+    rounded?: string,
+    outlined?: string,
+    elevation0: string,
+    elevation1: string,
```

Comment:
Can these be optional (?) props? It feels silly to do, but let's just add all 25 of these elevation rule names.